.github/workflows/ci.yml

```diff
@@ -60,7 +60,7 @@ jobs:
         run: deno upgrade --canary
 
       - name: Format
-        run: deno fmt --check
+        run: deno fmt --check --ignore=fs/testdata/json_invalid.json
```

Comment:
Why? We should strive not to add any ignores and we got away with adding any until now

Reply:
-> https://github.com/denoland/deno_std/pull/743